llama_stack/apis/common/type_system.py

```diff
@@ -91,15 +91,18 @@ class DialogType(BaseModel):
     name="ParamType",
 )
 
+"""
 # TODO: recursive definition of ParamType in these containers
 # will cause infinite recursion in OpenAPI generation script
 # since we are going with ChatCompletionInputType and CompletionInputType
 # we don't need to worry about ArrayType/ObjectType/UnionType for now
-# ArrayType.model_rebuild()
-# ObjectType.model_rebuild()
-# UnionType.model_rebuild()
+ArrayType.model_rebuild()
+ObjectType.model_rebuild()
+UnionType.model_rebuild()
 
 
-# class CustomType(BaseModel):
-#     type: Literal["custom"] = "custom"
-#     validator_class: str
+class CustomType(BaseModel):
+pylint: disable=syntax-error
```

Comment:
Why is this needed if all of this is inside a string? Why was it even needed to be changed from a comment to a quoted string?

Reply:
mypy flagged `type: Literal["custom"] = "custom"` as an issue, even though it was inside a comment. Wrapping the block in a docstring made it invisible to MyPy, preventing the error.